### ISSUE TITLE
Incompatible with Django 3.2 and python-binary-memcached<=0.29

### ISSUE THREAD

**Issue by DrMeers** [jaysonsantos/django-bmemcached#20]
- `has_key()` returns `True` for all keys.

Presumably something to do with https://github.com/django/django/commit/bb64b99b78a579cb2f6178011a4cf9366e634438 -- adding `_missing_key = None` makes it work correctly.

Was testing with `python-binary-memcached==0.29`, will try newer version and see if that supports default values in `get()`

**Comment by DrMeers** [comment]
`0.30` changelog says "Add ability to return default value on get but breaking get’s API" so maybe that will fix it with https://github.com/jaysonsantos/python-binary-memcached/commit/470b5dabf507e6c7a9d5485e46648840c303cd97

**Comment by DrMeers** [comment]
Confirmed it works with `python-binary-memcached>=0.30` -- not sure if something can be added to the dependency info to ensure people don't get bitten by the `django>=3.2` + `python-binary-memcached<=0.29` combo?

**Comment by jaysonsantos** [comment]
Hey there @DrMeers that is a hard one to tackle as people install this package and freeze the whole venv and do not bump the required library when needed.
I'd suggest using something like poetry that would know how to deal properly with this locking.
So, if you get a the current version of django-bmemcached and python-binary-memcached, does it work?

**Comment by DrMeers** [comment]
> So, if you get a the current version of django-bmemcached and python-binary-memcached, does it work?

Yep, confirmed above it works fine for `python-binary-memcached>=0.30`
